ci/cron/daily-compat.yml

```diff
@@ -370,7 +370,6 @@ jobs:
            && "$(perf_speedy)" == "Succeeded"
            && "$(check_releases)" == "Succeeded"
            && "$(update_canton)" == "Succeeded"
-           && "$(update_canton_2x)" == "Succeeded"
```

Comment:
This variable no longer exits so this does not get substituted by Azure Pipelines and Bash treats it as a failure (`update_canton_2x` is not an executable in `PATH`), but since we're in a Boolean evaluation context this just makes the entire expression "false" instead of failing the script.